packages/expo-screen-orientation/ios/EXScreenOrientation/EXScreenOrientationViewController.m

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019-present 650 Industries. All rights reserved.
+
+#import <UMCore/UMModuleRegistryProvider.h>
+
+#import <EXScreenOrientation/EXScreenOrientationViewController.h>
+#import <EXScreenOrientation/EXScreenOrientationRegistry.h>
+
+@interface EXScreenOrientationViewController ()
+
+@property (nonatomic, weak) RCTBridge *bridge;
+@property (nonatomic, assign) UIInterfaceOrientationMask defaultOrientationMask;
+
+@end
+
+@implementation EXScreenOrientationViewController
+
+- (instancetype)initWithBrigde:(RCTBridge*)brigde andDefaultScreenOrientationMask:(UIInterfaceOrientationMask)defaultOrientationMask
```

Comment:
```suggestion
- (instancetype)initWithBrigde:(RCTBridge *)bridge andDefaultScreenOrientationMask:(UIInterfaceOrientationMask)defaultOrientationMask
```

Reply:
Or, in fact, I don't see any use of `bridge` in this class. Removing its reference would be not only good given we don't use it, but also iirc multiple `RCTBridges` can be instantiated throughout lifetime of the app (isn't reloading recreating the bridge?) — if so, since `_bridge` isn't updated anywhere here and there is only one `ViewController` instantiated in a bare app we wouldn't have new instances of bridges here